packages/shared/src/BootcAPI.ts

```diff
@@ -0,0 +1,27 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ l***********************************************************************/
+
+import type { BootcBuildInfo } from './models/bootc';
+import type { ImageInfo } from '@podman-desktop/api';
+
+export abstract class BootcApi {
```

Comment:
why is it not an interface ?

Reply:
we implement it in: https://github.com/containers/podman-desktop-extension-bootc/pull/180/files#diff-19eddadfc844f5ed33879b5ac1ab1477398cdbf787d97812aa73a7c266cb3401R8

Reply:
yes but why it's not `export interface BootcApi`